src/Firebase/Database.php

```diff
@@ -76,10 +76,7 @@ public function getReferenceFromUrl($uri): Reference
         $uri = $uri instanceof UriInterface ? $uri : new Uri($uri);
 
         if (($givenHost = $uri->getHost()) !== ($dbHost = $this->uri->getHost())) {
-            throw new InvalidArgumentException(\sprintf(
-                'The given URI\'s host "%s" is not covered by the database for the host "%s".',
-                $givenHost, $dbHost
-            ));
+            throw new InvalidArgumentException(\sprintf('The given URI\'s host "%s" is not covered by the database for the host "%s".', $givenHost, $dbHost));
```

Comment:
Could you please revert the changes in this (unrelated) class? 🙈

Reply:
I think the php-cs fixer does _this fix_ :p 
I'll fix that too :)

Reply:
I was surprised as well because it doesn't on my machine™ 😅